srcpkgs/python3-pythondialog/template

```diff
@@ -1,14 +1,15 @@
 # Template file for 'python3-pythondialog'
 pkgname=python3-pythondialog
-version=3.5.1
-revision=4
+version=3.5.3
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 makedepends="python3-devel"
 depends="python3 dialog"
 short_desc="Python wrapper for the dialog utility"
 maintainer="svenper <svenper@tuta.io>"
 license="LGPL-2.1-or-later"
-homepage="http://pythondialog.sourceforge.net"
+homepage="https://pythondialog.sourceforge.io"
+changelog="https://sourceforge.net/p/pythondialog/code/ci/master/tree/ChangeLog.init?format=raw"
```

Comment:
I think https://pythondialog.sourceforge.io/news.html is more useful

Reply:
True, I think I missed that back when I added this link.